grade_test.go

```diff
@@ -261,14 +269,33 @@ func TestRunCompiled(t *testing.T) {
 		t.Errorf("Failed to open the compiled file: [err=%e]", err)
 	}
 
-	expected := "Hello world!\n"
-	actual := runCompiled(dir, "", []string{})
+	expected := "Hello, C Plus Plus!\n"
+	actual := runCompiled(dir, tmp.Name(), "c++", []string{"Hello, C Plus Plus\n"})
 
 	if expected != actual {
 		t.Errorf("Expected text did not match actual [expected=%#v] [actual=%#v]", expected, actual)
 	}
 }
 
+func TestRunInterpreted(t *testing.T){
+	js := `console.log("Hello, World!");`
+	tmp, _ := os.CreateTemp("", "*.js")
+	// Removes the created temp file, this will always run
+	defer os.Remove(tmp.Name())
+	
+	tmp.WriteString(js)
+	parts := strings.Split(tmp.Name(), "/")
+	parts = parts[:len(parts)-1]
+	
+	dir := strings.Join(parts, "/")
+	expected := "Hello, World!\n"
+	actual := runInterpreted(dir, tmp.Name(), "javascript", []string{"Hello, World!"})
```

Comment:
arg pass doesn't make sense

Reply:
Again, I think I completely misunderstood that last argument. Going to fix.